src/server/api/routers/dbquery.ts

```diff
@@ -0,0 +1,33 @@
+import { TRPCError } from "@trpc/server";
+import { z } from "zod";
+
+import {
+  createTRPCRouter,
+  protectedProcedure,
+  publicProcedure,
+} from "~/server/api/trpc";
+
+import { db } from "~/server/db";
+
+export const dbQueryRouter = createTRPCRouter({
```

Comment:
Can we name this `userRouter` and name the file `user.ts`? We are going to add a lot more routers for different parts of the application, so naming this dbQueryRouter doesn't make much sense. Name it on functionality not implementation.

Reply:
Any comments on using protected procedure rather than public procedure?